lib/matplotlib/cbook.py

```diff
@@ -21,8 +21,14 @@
 import types
 import weakref
 
+from packaging.version import parse as parse_version
 import numpy as np
 
+if parse_version(np.__version__) >= parse_version("1.25.0"):
+    from numpy.exceptions import VisibleDeprecationWarning
+else:
+    from numpy import VisibleDeprecationWarning
```

Comment:
I'd rather make this implicit and not parse versions

```suggestion
try:
    from numpy.exceptions import VisibleDeprecationWarning  # numpy >= 1.25
except ImportError:
    from numpy import VisibleDeprecationWarning
```